saleor/discount/utils.py

```diff
@@ -1002,8 +1002,16 @@ def _get_defaults_for_gift_line(
             "currency": order_or_checkout.currency,
         }
     else:
+        variant = (
+            ProductVariant.objects.filter(id=variant_id)
+            .select_related("product")
+            .only("sku", "product__name")
+            .first()
+        )
```

Comment:
Could we pass that info from `LinesInfo` objects instead of fetching them again?

Reply:
This line doesn't exists, when we are fetching the lines. This is needed to create a new order line (gift).